jenkins/databricks/build.sh

```diff
@@ -80,6 +80,7 @@ initialize()
         cd spark-rapids
     fi
 
+    git submodule update --init
```

Comment:
Same thing here.  This script is run by developers that aren't necessarily going to expect it to make changes to source.

Reply:
Make sense, let me move the the `git submodule update --init` just under where we checkout spark-rapids.git